packages/installer.vm/installer.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>installer.vm</id>
-    <version>0.0.0.20240304</version>
+    <version>0.0.0.20240305</version>
```

Comment:
we are in the future :rocket:

Reply:
so can i just use the installer.ps1 script again on the read me page? or do i have to do something special now?

Reply:
you can just use the installer.ps1 in the flare-vm repo. It will take the latest package from myget